nodes/core/locales/en-US/messages.json

```diff
@@ -332,24 +333,28 @@
             "verify-server-cert":"Verify server certificate",
             "compatmode": "Use legacy MQTT 3.1 support"
         },
+        "sections-label":{
+            "birth-message": "Message sent on connection (birth message)",
+            "will-message":"Message sent on an unexpected disconnection (will message)",
+            "close-message":"Message sent before disconnecting (close message)"
+        },
         "tabs-label": {
             "connection": "Connection",
             "security": "Security",
-            "will": "Will Message",
-            "birth": "Birth Message"
+            "messages": "Messages"
         },
         "placeholder": {
             "clientid": "Leave blank for auto generated",
             "clientid-nonclean":"Must be set for non-clean sessions",
             "will-topic": "Leave blank to disable will message",
-            "birth-topic": "Leave blank to disable birth message"
+            "birth-topic": "Leave blank to disable birth message",
+            "close-topic": "Leave blank to disable close message"
         },
         "state": {
             "connected": "Connected to broker: __broker__",
             "disconnected": "Disconnected from broker: __broker__",
             "connect-failed": "Connection failed to broker: __broker__"
         },
-        "retain": "Retain",
```

Comment:
This message is used by the `mqtt out` node so cannot be removed.

Reply:
sorry - will add back

Reply:
Yes, please go ahead and fix the css for the fields post merge.  thanks Nick